src/backend/graph.test.js

```diff
@@ -87,14 +87,36 @@ describe("graph", () => {
         .addEdge(crabIngredientEdge())
         .addEdge(eatEdge());
 
+    const crabLoop = () => ({
+      address: makeAddress("crab-self-assessment"),
+      src: crabNode().address,
+      dst: crabNode().address,
+      payload: {evaluation: "not effective at avoiding hero"},
+    });
+
+    const critCookEdge = () => ({
```

Comment:
I'd like to see a comment/name here indicating what value this provides: namely, it tests the case where we have non-singleton hom-sets (i.e., there are multiple arrows from `a` to `b`).